tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -141,6 +141,13 @@ sub run {
         # We expect hostnames reported by terraform to match the actual hostnames in Azure and GCE
         die "Expected hostname $expected_hostname is different than actual hostname [$real_hostname]"
           if ((is_azure() || is_gce()) && ($expected_hostname ne $real_hostname));
+        if (get_var('FENCING_MECHANISM') eq 'native' && get_var('PUBLIC_CLOUD_PROVIDER') eq 'AZURE') {
+            qesap_az_setup_native_fencing_permissions(
```

Comment:
Does not `qesap_az_setup_native_fencing_permissions` return anything to the user? Is it a list of `az` commands to query and set?

Reply:
Not sure if there is anything to return here. 
'az vm identity assign' returns identity id which is checked within the library. If identity ID is not valid, function croaks. So here it is only about function to execute command and pass.

https://github.com/os-autoinst/os-autoinst-distri-opensuse/compare/master...lpalovsky:os-autoinst-distri-opensuse:Azure-MSI-based-fencing#diff-23c988d39d71dc252bb5dce95a699047e93941078f2152ab4a6587286e9a4e47R1866